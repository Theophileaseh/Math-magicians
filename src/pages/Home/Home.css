.maincontainer {
  padding: 30px;
  padding-top: 120px;
  background: #1a011a;
  height: 100vh;
}

.intro {
  font-weight: 400;
  color: #fff;
}
