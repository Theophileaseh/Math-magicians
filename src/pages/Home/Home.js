import React from 'react';
import './Home.css';

const Home = () => (
  <div className="maincontainer">
    <h2 className="intro">Mathematics is not about number, equations, computations, or algorithms: it is about understanding. William Paul Thurston</h2>
  </div>
);

export default Home;
