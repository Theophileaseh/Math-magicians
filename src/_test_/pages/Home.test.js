import React from 'react';
import renderer from 'react-test-renderer';

import Home from '../../pages/Home/Home';

it('matches home snapshot', () => {
  const tree = renderer.create(
    <Home />,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
