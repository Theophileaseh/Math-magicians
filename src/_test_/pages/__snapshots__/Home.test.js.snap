// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches home snapshot 1`] = `
<div
  className="maincontainer"
>
  <h2
    className="intro"
  >
    Mathematics is not about number, equations, computations, or algorithms: it is about understanding. William Paul Thurston
  </h2>
</div>
`;
